src/ontology/cl-edit.owl

```diff
@@ -21914,6 +21915,16 @@ AnnotationAssertion(rdfs:label obo:CL_0011016 "flagellated sperm cell")
 EquivalentClasses(obo:CL_0011016 ObjectIntersectionOf(obo:CL_0011013 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0030317)))
 SubClassOf(obo:CL_0011016 obo:CL_0011013)
 
+# Class: obo:CL_0011017 (renal tubular epithelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "http://www.cloud-clone.com/items/I061.html") obo:IAO_0000115 obo:CL_0011017 "A layer of cells in the outer layer of the renal tubule.")
```

Comment:
Should the definition of a cell type start with the phrase "a layer of cells"?

Reply:
That sounds like a definition for a related Uberon tissue term.   Text definition should be something like

"An epithelial cell fo the outer layer of the renal tubule'  - however, I wonder if that is specific enough - see other comments.